test/Dapr.Actors.AspNetCore.IntegrationTest/ActivationTests.cs

```diff
@@ -50,7 +50,7 @@ private async Task<string> IncrementCounterAsync(HttpClient httpClient, string a
             var actorTypeName = nameof(DependencyInjectionActor);
             var methodName = nameof(DependencyInjectionActor.IncrementAsync);
 
-            var request = new HttpRequestMessage(HttpMethod.Put, $"http://localhost/actors/{actorTypeName}/{actorId}/method/{methodName}");
+            var request = new HttpRequestMessage(HttpMethod.Put, $"http://localhost:65326/actors/{actorTypeName}/{actorId}/method/{methodName}");
```

Comment:
What's the reason for this change?

Reply:
I added that when I was debugging the cookie issue. I'll remove it.